packages/venia-concept/webpack.config.js

```diff
@@ -1,261 +1,42 @@
-            new webpack.EnvironmentPlugin(projectConfig.env),
-            new ServiceWorkerPlugin({
-                mode,
-                paths: themePaths,
-                injectManifest: true,
-                injectManifestConfig: {
-                    include: [/\.js$/],
-                    swSrc: './src/sw.js',
-                    swDest: 'sw.js'
-                }
-            }),
-            new WebpackAssetsManifest({
-                output: 'asset-manifest.json',
-                entrypoints: true,
-                publicPath: '/',
-                // Add explicit properties to the asset manifest for
-                // venia-upward.yml to use when evaluating app shell templates.
-                transform(assets) {
-                    // All RootComponents go to prefetch, and all client scripts
-                    // go to load.
-                    assets.bundles = {
-                        load: assets.entrypoints.client.js,
-                        prefetch: []
-                    };
-                    Object.entries(assets).forEach(([name, value]) => {
-                        if (name.startsWith('RootCmp')) {
-                            const filenames = Array.isArray(value)
-                                ? value
-                                : [value];
-                            assets.bundles.prefetch.push(...filenames);
-                        }
-                        const ext = path.extname(name);
-                        const type = ext && ext.replace(/^\./, '');
-                        if (type) {
-                            if (!assets[type]) {
-                                assets[type] = {};
-                            }
-                            assets[type][path.basename(name, ext)] = value;
-                        }
-                    });
-                }
-            })
+const { configureWebpack } = require('@magento/pwa-buildpack');
+
+module.exports = async env => {
+    const config = await configureWebpack({
+        context: __dirname,
+        vendor: [
+            'apollo-cache-inmemory',
+            'apollo-cache-persist',
+            'apollo-client',
+            'apollo-link-context',
+            'apollo-link-http',
+            'informed',
+            'react',
+            'react-apollo',
+            'react-dom',
+            'react-feather',
+            'react-redux',
+            'react-router-dom',
+            'redux',
+            'redux-actions',
+            'redux-thunk'
         ],
-        optimization: {
-            splitChunks: {
-                cacheGroups: {
-                    vendor: {
-                        test: new RegExp(
-                            `[\\\/]node_modules[\\\/](${libs.join('|')})[\\\/]`
-                        ),
-                        chunks: 'all'
-                    }
-                }
+        special: {
+            '@magento/peregrine': {
+                esModules: true,
+                cssModules: true
```

Comment:
```suggestion
```

There is no CSS in `@magento/peregrine` today, and we have no plans to add any to it, so I believe this flag can be removed.